samples/server/petstore/aspnetcore-8.0-nullableReferenceTypes/src/Org.OpenAPITools/Models/NullableClass.cs

```diff
@@ -0,0 +1,279 @@
+ *
+ * The version of the OpenAPI document: 1.0.0
+ * 
+ * Generated by: https://openapi-generator.tech
+ */
+
+using System;
+using System.Linq;
+using System.Text;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.Runtime.Serialization;
+using Newtonsoft.Json;
+using Org.OpenAPITools.Converters;
+
+namespace Org.OpenAPITools.Models
+{ 
+    /// <summary>
+    /// 
+    /// </summary>
+    [DataContract]
+    public partial class NullableClass : IEquatable<NullableClass>
+    {
+        /// <summary>
+        /// Gets or Sets IntegerProp
+        /// </summary>
+        [DataMember(Name="integer_prop", EmitDefaultValue=true)]
+        public int? IntegerProp { get; set; }
+
+        /// <summary>
+        /// Gets or Sets NumberProp
+        /// </summary>
+        [DataMember(Name="number_prop", EmitDefaultValue=true)]
+        public decimal? NumberProp { get; set; }
+
+        /// <summary>
+        /// Gets or Sets BooleanProp
+        /// </summary>
+        [DataMember(Name="boolean_prop", EmitDefaultValue=true)]
+        public bool? BooleanProp { get; set; }
+
+        /// <summary>
+        /// Gets or Sets StringProp
+        /// </summary>
+        [DataMember(Name="string_prop", EmitDefaultValue=true)]
+        public string? StringProp { get; set; }
+
+        /// <summary>
+        /// Gets or Sets DateProp
+        /// </summary>
+        [DataMember(Name="date_prop", EmitDefaultValue=true)]
+        public DateOnly? DateProp { get; set; }
+
+        /// <summary>
+        /// Gets or Sets DatetimeProp
+        /// </summary>
+        [DataMember(Name="datetime_prop", EmitDefaultValue=true)]
+        public DateTime? DatetimeProp { get; set; }
+
+        /// <summary>
+        /// Gets or Sets ArrayNullableProp
+        /// </summary>
+        [DataMember(Name="array_nullable_prop", EmitDefaultValue=true)]
+        public List<Object> ArrayNullableProp { get; set; }
+
+        /// <summary>
+        /// Gets or Sets ArrayAndItemsNullableProp
+        /// </summary>
+        [DataMember(Name="array_and_items_nullable_prop", EmitDefaultValue=true)]
+        public List<Object?> ArrayAndItemsNullableProp { get; set; }
+
+        /// <summary>
+        /// Gets or Sets ArrayItemsNullable
+        /// </summary>
+        [DataMember(Name="array_items_nullable", EmitDefaultValue=false)]
+        public List<Object?> ArrayItemsNullable { get; set; }
+
+        /// <summary>
+        /// Gets or Sets ObjectNullableProp
+        /// </summary>
+        [DataMember(Name="object_nullable_prop", EmitDefaultValue=true)]
+        public Dictionary?<string, Object> ObjectNullableProp { get; set; }
+
+        /// <summary>
+        /// Gets or Sets ObjectAndItemsNullableProp
+        /// </summary>
+        [DataMember(Name="object_and_items_nullable_prop", EmitDefaultValue=true)]
+        public Dictionary?<string, Object?> ObjectAndItemsNullableProp { get; set; }
```

Comment:
nullable reference is not working correctly when enabled on dictionary. From current implementation, it should be treated as `Dictionary<string, Object?>` though subsequently it should be fixed to address as `Dictionary<string, Object?>?`